PSKoans/Init/RegisterArgumentCompleters.ps1

```diff
@@ -45,3 +47,28 @@ $RegisterParams.ParameterName = 'IncludeModule'
 Register-ArgumentCompleter @RegisterParams
 
 #endregion
+
+#region Name completer
+
+$RegisterParams = @{
+    CommandName   = $CommandName
+    ParameterName = 'Name'
+    ScriptBlock   = {
+        param($Command, $Parameter, $WordToComplete, $CommandAst, $FakeBoundParams)
+
+        $Values = Get-Module PSKoans |
+            Select-Object -ExpandProperty ModuleBase |
```

Comment:
Not a bad way to go, but you can target `$script:ModuleRoot` to get this path as this file is dot-sourced into the psm1. :slightly_smiling_face:

Reply:
I'll change this tomorrow!